website/docs/r/netapp_backup_policy.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+subcategory: "NetApp"
+layout: "azurerm"
+page_title: "Azure Resource Manager: netapp_backup_policy"
+description: |-
+  Manages a NetApp Backup Policy.
+---
+
+# netapp_backup_policy
+
+Manages a NetApp Backup Policy.
+
+## NetApp Backup Policy Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_netapp_account" "example" {
+  name                = "example-netappaccount"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_netapp_backup_policy" "example" {
+  name                    = "example-netappbackuppolicy"
+  resource_group_name     = azurerm_resource_group.example.name
+  location                = azurerm_resource_group.example.location
+  account_name            = azurerm_netapp_account.example.name
+  daily_backups_to_keep   = 2
```

Comment:
We can remove these Optional attributes as they have defaults and don't need to be specified when creating this resource